### ISSUE TITLE
Do you have model zoo?

### ISSUE THREAD

**Issue by poodarchu** [facebookresearch/ClassyVision#472]

**Comment by mannatsingh** [comment]
@poodarchu We don't have a model zoo at the moment, although we're actively considering our options. Classy Vision supports using any PyTorch model, so if you have access to any model zoo (like torchvision), you can use those models with Classy Vision!

**Comment by poodarchu** [comment]
but if you donnot have a model zoo, then I will be not sure about performance if there's any mismatch between your implentations and what reported in their papers.

**Comment by mannatsingh** [comment]
@poodarchu we plan on releasing training recipes along with the expected accuracy of our models, which should be reproducible. I am also planning on adding a tutorial so that you can use your favorite PyTorch pretrained models with Classy Vision too!

**Comment by poodarchu** [comment]
great.
